setup.cfg

```diff
@@ -26,6 +26,10 @@ classifier =
 packages =
     scality_manila_utils
 
+[entry_points]
+console_scripts =
+    manila-cli = scality_manila_utils.cli:main
```

Comment:
Better call this `scality-manila-utils` or something alike, `manila-cli` could be from the upstream Manila project.